### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by timobrembeck**

### CODE REVIEW

File: js/pokemap.js [hunk 1]
```diff
@@ -4,87 +4,53 @@ var util = require('util');
...
-var getPokemonById = "/api/pokemon/id/";
+var apiEndpoint=null;
+var getAllPokemon = "pokemon/";
+var getAllSightings = getAllPokemon + "sighting/";
+var getAllSightingsByTime = getAllSightings + "ts/";
+var getPokemonById = getAllPokemon + "id/";
 var getAllPredictions = {};
 
-var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndPoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014'}) {
+var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndpoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014/api/'}) {
```
Comment (johartl) [inline js/pokemap.js]: This is incorrect. The `apiEndpoint` will default to `http://pokedata.c4e3f8c7.svc.dockerapp.io:65014` so you have to append the `/api` yourself where necessary.

Comment (timobrembeck) [reply js/pokemap.js]: Thanks!

**Comment by johartl** [comment]
Will this be merged eventually?

**Comment by farisca** [comment]
Yes, and also the new npm package will be published in a few seconds.

**Pull request merged**